homeassistant/components/wirelesstag/__init__.py

```diff
@@ -126,6 +128,20 @@ def push_callback(tags_spec, event_spec):
         self.api.start_monitoring(push_callback)
 
 
+def async_migrate_unique_id(hass: HomeAssistant, tag: SensorTag, domain: str, key: str):
```

Comment:
Please add a type annotation for the return value. We want to type the whole signature when adding type annotations.

```py
..., key: str) -> None:
```

Reply:
Addressed in [new PR](https://github.com/home-assistant/core/pull/104828).